### PR TITLE
New package: yaru-r28

### PR DESCRIPTION
**Pull request by dawidpotocki**
[Yaru](https://github.com/ubuntu/yaru) is a GTK and icon theme used by default in Ubuntu 18.10+. I personally need it to check how my programs would look under Ubuntu. I hope that I made template file right :P.

### CODE REVIEW

File: srcpkgs/yaru/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'yaru'
+pkgname=yaru
+version=r28
+revision=1
+archs=noarch
+wrksrc="yaru-$version"
+build_style=meson
+makedepends="glib-devel gtk+3-devel meson sassc"
```
Comment (maxice8) [inline srcpkgs/yaru/template]: glib-devel is a package for tooling while building packages (like glib-mkenums) and it should
be in hostmakedepends=

gtk+3-devel is the library so it should stay in makedepends=

build_style=meson already adds a dependency on meson so you don't need to define it again

sassc is an executable and should go to hostmakedepends=

Please read up on the different types of depends https://github.com/void-linux/void-packages/blob/master/Manual.md#explain_depends

File: srcpkgs/yaru/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'yaru'
+pkgname=yaru
+version=r28
+revision=1
+archs=noarch
+wrksrc="yaru-$version"
```
Comment (maxice8) [inline srcpkgs/yaru/template]: no need to do this, this is the default value of `wrksrc` `${pkgname}-${version}`

**Comment by dawidpotocki** [comment]
Oh, I read some of that xbps-src manual, but I missed it and I was actually looking for it as I didn't understand the difference between these deps. So I guess that it is correct now?

**Comment by maxice8** [comment]
looking at the travis logs:

```

Found Pkg-config: NO
WARNING: Could not detect glib version, assuming 2.54. You may get build errors if your glib is older.
```

while Void Linux tracks latest glib, i think it is better we provide the correct version so we need `pkg-config` on hostmakedepends= and `libglib-devel` on makedepends=


After that is done i think it is good to go.

**Comment by maxice8** [comment]
Thanks for your contribution.

Please notify (comment) when proposed changes are done.

**Pull request merged**